Source/WebCore/bindings/js/JSWindowProxy.h

```diff
@@ -64,8 +64,7 @@ class WEBCORE_EXPORT JSWindowProxy final : public JSC::JSGlobalProxy {
     Ref<DOMWindow> protectedWrapped() const;
     static WindowProxy* toWrapped(JSC::VM&, JSC::JSValue);
 
-    DOMWrapperWorld& world();
-    Ref<DOMWrapperWorld> protectedWorld();
+    DOMWrapperWorld& world() const;
```

Comment:
Does this need to be `const`? It's a little odd to have the function `const` when it's not returning a `const` reference.

Reply:
Thanks, I'll look into this a bit more. Marking this as draft for now. I couldn't make it inline easily given:

> Inline functions should not be in classes annotated with WEBCORE_EXPORT. Remove the macro from the class and apply it to each appropriate method, or move the inline function definition out-of-line.

Reply:
> Inline functions should not be in classes annotated with WEBCORE_EXPORT. Remove the macro from the class and apply it to each appropriate method, or move the inline function definition out-of-line.

I would ignore that. There are other inline functions in this class already anyway.